pandas/tests/test_categorical.py

```diff
@@ -1175,6 +1175,17 @@ def test_comparison_with_unknown_scalars(self):
         self.assert_numpy_array_equal(cat == 4 , [False, False, False])
         self.assert_numpy_array_equal(cat != 4 , [True, True, True])
 
+    def test_shift(self):
+        # GH10495
+        # Series.shift should not depend on the dtype being categorical or not
+        values = [['a', 'b', 'c']]
+        shifts = [-1, 0, 1]
+        for v in values:
+            for shift in shifts:
+                a = pd.Series(v).shift(shift)
+                b = pd.Series(v, dtype='category').shift(shift)
```

Comment:
this needs to assert that the result series is equal 
so construct the expected result as series